src/js/angular/core/templates/search-resource-input.html

```diff
@@ -26,6 +26,12 @@
     </span>
 </div>
 <span class="autocomplete-hint">Hint: "ab c" matches "abC*", "ab c*" and "ab-c*"</span>
+<div ng-if="exampleValue">
```

Comment:
I think if example value is present, we may show it after the text: "Search for an IRI to view it. This is a user defined advanced configuration. Hint: " in graphs-visualizations.html